### ISSUE TITLE
Color Contrast Problems with Default Colors

### ISSUE THREAD

**Issue by mgifford** [fullcalendar/fullcalendar#4239]
### Bug Reports

Image colors don't meet WCAG 2.0 AA color contrast with defaults in demo https://fullcalendar.io or default install.

<img width="1092" alt="screen shot 2018-07-05 at 10 34 34 pm" src="https://user-images.githubusercontent.com/116832/42357018-b60ede62-80a3-11e8-8227-f6a3fb488ae7.png">

Color contrast fail #3a87ad / #fff - http://contrast-ratio.com/#%233a87ad-on-white

This is an issue which should be easy to resolve.

A close color can be chosen from http://contrast-finder.tanaguru.com/ which meets the requirements for contrast.
